### PR TITLE
Start in documenting with ReadTheDocs

### LINKED ISSUE #130: Provide a first frame for documentation
Currently, there is a good lack of documentation. Most favorably introduce 'Read the docs' for documentation purposes.

### PR DESCRIPTION
**Pull request by ckittl**
Resolves #130

### CODE REVIEW

File: docs/readthedocs/models/input/grid/node.rst [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
++---------------+------+--------------------------------------------------------------------+
+| voltLvl       | --   | Information of the voltage level (id and nominal voltage)          |
++---------------+------+--------------------------------------------------------------------+
+| subnet        | --   | Sub grid number                                                    |
++---------------+------+--------------------------------------------------------------------+
+
+Caveats
+^^^^^^^
+System participants, that need to have geographical locations, inherit the position from the node.
+If the overall location does not play a too big role, you are able to use the default location with
```
Comment (sensarmad) [inline docs/readthedocs/models/input/grid/node.rst:35]: play a big role

File: docs/readthedocs/models/input/grid/nodegraphic.rst [hunk 2]
```diff
@@ -0,0 +1,27 @@
+.. _node_graphic_model:
+
+Schematic Node Graphic
+----------------------
+Schematic drawing information for a line model.
```
Comment (sensarmad) [inline docs/readthedocs/models/input/grid/nodegraphic.rst:5]: node model

File: docs/readthedocs/models/input/grid/switch.rst [hunk 3]
```diff
@@ -0,0 +1,40 @@
+.. _switch_model:
+
+Switch
+------
+Whoops!
```
Comment (sensarmad) [inline docs/readthedocs/models/input/grid/switch.rst:5]: Oopsie! :D

File: docs/readthedocs/models/input/participant/general.rst [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Reactive power characteristics are designed to describe reactive power control behaviour of the models.
+In Germany, system operators can require system participants to follow certain characteristics specified in the
+operators technical requirements and individually selected per connected asset.
+
+Currently three different characteristics are implemented:
+
+Fixed Power Factor
+""""""""""""""""""
+Active and reactive power are coupled by a timely independent power factor.
```
Comment (sensarmad) [inline docs/readthedocs/models/input/participant/general.rst:16]: time-independent

File: docs/readthedocs/models/input/participant/storage.rst [hunk 5]
```diff
@@ -0,0 +1,77 @@
+.. _storage_model:
+
+Electrical Energy Storage
+-------------------------
+Whoops!
+Seems, you found a construction site...
```
Comment (sensarmad) [inline docs/readthedocs/models/input/participant/storage.rst:6]: oopsie!

File: docs/readthedocs/models/input/participant/wec.rst [hunk 6]
```diff
@@ -0,0 +1,75 @@
+.. _wec_model:
+
+Wind Energy Converter
+---------------------
+Whoops!
```
Comment (sensarmad) [inline docs/readthedocs/models/input/participant/wec.rst:5]: oopsie!

File: docs/readthedocs/models/models.rst [hunk 7]
```diff
@@ -0,0 +1,170 @@
...
+   result/participant/load
+   result/participant/pv
+   result/participant/storage
+   result/participant/wec
+   result/participant/thermalsink
+   result/participant/thermalstorage
+   result/participant/thermalunit
+   result/participant/thermalhouse
+   result/participant/cylindricalstorage
+   result/participant/systemparticipant
```
Comment (sensarmad) [inline docs/readthedocs/models/models.rst:122]: please add hp

File: docs/readthedocs/models/models.rst [hunk 8]
```diff
@@ -0,0 +1,170 @@
...
+   Each time instance in this time series has its own value (random duplicates may occur obviously).
+   They are only applicable for the time frame that is defined by the content of the time series.
+
+.. _repetitive_time_series:
+
+Repetitive Time Series
+   Those time series do have repetitive values, e.g. each day or at any other period.
+   Therefore, they can be applied to any time frame, as the mapping from time instant to value is made by information
+   reduction.
+   Despite the actual data, also a mapping function has to be known.
```
Comment (sensarmad) [inline docs/readthedocs/models/models.rst:143]: In addition to actual data, a mapping function has to be known.

File: docs/uml/main/DataSourceClassDiagramm_draft.puml [hunk 9]
```diff
@@ -0,0 +1,126 @@
+@startuml
+
+note "Assuming all classes to implement \nthe abstract methods of their interfaces\n\n" as generalNotes
+
+interface Publisher
```
Comment (sensarmad) [inline docs/uml/main/DataSourceClassDiagramm_draft.puml:5]: do we still need this draft?

File: docs/readthedocs/models/models.rst (hunk 7 above)
Comment (sensarmad) [reply docs/readthedocs/models/models.rst:122]: As I did the result part, I could do this as well. Please let me know

File: docs/uml/main/DataSourceClassDiagramm_draft.puml (hunk 9 above)
Comment (ckittl) [reply docs/uml/main/DataSourceClassDiagramm_draft.puml:5]: Wasn't sure, because @mia-krause is partly working on it. Therefore, I would like to keep it.

**Pull request merged**